index.html

```diff
@@ -41,6 +40,151 @@ <h1>I'm Tash <br> Glad to see you!</h1>
 			    </div>
 			</div>
 		</section>
+
+		<section class="works">
+			<div class="works1">
+			<img class="snap" src="images/Snap3.png" alt="">
+			<p class="tonic">Tonic</p>
```

Comment:
- [ ] Kindly use a `<h2>` or `<h3>` tag for your card titles instead of a `p` tag since they're basically headings and it would be a better semantic representation of the element.